src/core/Akka.Tests/Actor/ActorRefSpec.cs

```diff
@@ -73,6 +74,7 @@ public void An_ActorRef_should_not_equal_another_ActorRef_when_path_differs()
         public void An_ActorRef_should_not_allow_actors_to_be_created_outside_an_ActorOf()
         {
             Shutdown();
+            InternalCurrentActorCellKeeper.Current = null;
```

Comment:
I believe so. Something weird is happening with the threads and i noticed a few ThreadStatic attributes in the codebase.